engine/src/transaction/eip_1559.rs

```diff
@@ -0,0 +1,129 @@
+use crate::prelude::{Vec, U256};
+use crate::transaction::eip_2930::AccessTuple;
+use aurora_engine_types::types::Wei;
+use ethabi::Address;
+use rlp::{Decodable, DecoderError, Encodable, Rlp, RlpStream};
+
+/// Type indicator (per EIP-1559)
+pub const TYPE_BYTE: u8 = 0x01;
```

Comment:
Should be `0x02`?

Reply:
Would need to double check, I do recall pulling that off of source.